### DISCUSSION TITLE
[Support]: Recommended Installation

### DISCUSSION THREAD

**Discussion by MHKnowles** [blakeblackshear/frigate#13674] [General]
Hi there,
I'd like to use Frigate alongside my existing HomeAssistant implementation.  I currently use HA in a VMware ESXi virtual machine and have a few other VMs hosted there as well.  I am a Windows support guy by trade but can tackle Linux etc if there's a decent enough guide (I know enough to be dangerous, but not to troubleshoot incompatibilities etc).
I have bought a Google Coral USB and can pass it through to a VM if necessary.  However, the first guide I Googled appeared to say that Frigate either sits in Docker or you can build a Linux VM for it.  I went the latter route as i've never used Docker before, but quickly came to the realisation that if I used the latest Linux images (Ubuntu 24.04LTS in my case) they contained a newer version of Python than was necessary to do something with the Google Coral...and that's where i'm currently at.

Should I instead move to a Docker implementation?
Are there any long-term implications for support of a Google Coral with this small window of compatibility with obsolescent Python versions?  Does Docker get around this?

In short, what is the most popular and supportable way of getting Frigate up, running, and interacting with my HomeAssistant given I have a Coral USB and hardware that sits on ESXi?

**Comment by hawkeye217** [comment]
Docker is recommended. If you haven't seen the Getting Started Guide, check it out: https://docs.frigate.video/guides/getting_started

Also, there are particulars in the official documentation for ESXi. Check it out here: https://docs.frigate.video/frigate/installation/#esxi

**Reply by MHKnowles** [reply]
Yes I saw the page linking the ESXi documentation and tried to follow it due to my lack of Docker experience, which is when I came to the issue that the documentation doesn't work on current (24.04) Linux due to the Python issue.
